<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentArca</class>
 <widget class="QDialog" name="PaymentArca">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>284</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>284</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Arca Payment Method</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">/*background-image: url(/home/user/Desktop/is.png)</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="Ok_Cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ExpDate">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
color: rgb(255, 255, 255);
font: 11pt &quot;FreeMono&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>00/00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;FreeMono&quot;;;
color: rgb(255, 255, 255);
background-color: Transparent;</string>
   </property>
   <property name="text">
    <string>Exp.Date D/M</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/images/is.png);</string>
   </property>
   <widget class="QLineEdit" name="Name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(255, 255, 255);
font: 11pt &quot;FreeMono&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>NAME USERNAME</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="CardNumber">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
font: 13pt &quot;FreeMono&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>0000 0000 0000 0000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Cvv">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
font: 11pt &quot;FreeMono&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>000</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: Transparent;
font: 13pt &quot;FreeMono&quot;;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CARD NUMBER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: Transparent;
font: 13pt &quot;FreeMono&quot;;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CVV</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>label_2</zorder>
  <zorder>Ok_Cancel</zorder>
  <zorder>ExpDate</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Ok_Cancel</sender>
   <signal>accepted()</signal>
   <receiver>payment_arca</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Ok_Cancel</sender>
   <signal>rejected()</signal>
   <receiver>payment_arca</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
